<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Kvadrat tenglama</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">QLabel{
color:rgb(0, 0, 191);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
           </property>
           <property name="text">
            <string>c:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel{
color:rgb(0, 0, 191);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
           </property>
           <property name="text">
            <string>b:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel{
color:rgb(0, 0, 191);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
           </property>
           <property name="text">
            <string>a:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>natija:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Chiziqli tenglama</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">QLabel{
color:rgb(0, 0, 191);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
           </property>
           <property name="text">
            <string>a:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">QLabel{
color:rgb(0, 0, 191);
	font: 75 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
           </property>
           <property name="text">
            <string>b:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>natija:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
